src/Analysis/Ast/Impl/Analyzer/PythonAnalyzer.cs

```diff
@@ -205,12 +221,12 @@ public Task WaitForCompleteAnalysisAsync(CancellationToken cancellationToken = d
         internal void RaiseAnalysisComplete(int moduleCount, double msElapsed) 
             => AnalysisComplete?.Invoke(this, new AnalysisCompleteEventArgs(moduleCount, msElapsed));
 
-        private void AnalyzeDocument(AnalysisModuleKey key, PythonAnalyzerEntry entry, ImmutableArray<AnalysisModuleKey> dependencies) {
+        private void AnalyzeDocument(in AnalysisModuleKey key, in PythonAnalyzerEntry entry, in ImmutableArray<AnalysisModuleKey> dependencies) {
             _analysisCompleteEvent.Reset();
             ActivityTracker.StartTracking();
             _log?.Log(TraceEventType.Verbose, $"Analysis of {entry.Module.Name}({entry.Module.ModuleType}) queued");
 
-            var graphVersion = _dependencyResolver.ChangeValue(key, entry, entry.IsUserModule, dependencies);
+            var graphVersion = _dependencyResolver.ChangeValue(key, entry, entry.IsUserOrBuiltin || key.IsLibraryAsDocument, dependencies);
```

Comment:
Since library module is added to the graph as a separate entry, it doesn't have any references. So we need to add it as root to trigger analysis.